### PR TITLE
Find and add standard asset types

### LINKED ISSUE #642: Find and add standard asset types
It would be great to have a little collection of typical asset types that can be used by one if a simulation run. This issue collects the points necessary to do so:

- [ ] Find typical - at the moment - used types of each asset (Lines/Cables, WEC, Transformer, ...)  which is included in PSDM/SIMONA
- [ ] Identify sources for the input parameter of these to have a reference.
- [ ] Summarize everthing in read the docs and give some example files
- [ ] ...feel free to extend this, if necessary...

### PR DESCRIPTION
**Pull request by staudtMarius**
Resolves #642

### CODE REVIEW

**Review by sebastian-peter — DISMISSED** [review]
A proper review has to happen still, this is just a hint to add a meaningful message about what was added to the _changelog_  :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `ThermalGrid` as a container for a completely connected thermal grid
 - `EmResult` and `FlexOptionsResult` for Energy Management Systems [#651](https://github.com/ie3-institute/PowerSystemDataModel/issues/651)
 - `EvcsInput` now has a parameter for enabling and disabling vehicle to grid support [#681](https://github.com/ie3-institute/PowerSystemDataModel/issues/681)
+- Some standard asset types [#642](https://github.com/ie3-institute/PowerSystemDataModel/issues/642)
```
Comment (danielfeismann) [inline CHANGELOG.md:20]:
```suggestion
- Added some standard asset types to documenation [#642](https://github.com/ie3-institute/PowerSystemDataModel/issues/642)
```

**Review by danielfeismann — CHANGES_REQUESTED** [review]
This looks good to me, thanks for your work. Just a minor change I would suggest to change in the changelog.

**Review by danielfeismann — APPROVED** [review]
Thanks for improving this!

**Pull request merged**